js/utils/helpers.js

```diff
@@ -1938,6 +1938,19 @@
                                         }
 
                                         // handles `style="some: style;"` case from template
+                                        // min/max width needs to be applied to the wrapper of the caption and fullscreen button for consistent button placement
+                                        // check for min-width style
+                                        var minWidthIdx = attr[1].indexOf("min-width");
+                                        if (minWidthIdx >= 0) {
+                                            widthStyles += attr[1].substring(minWidthIdx, attr[1].indexOf(";", minWidthIdx)+1);
+                                        }
+
+                                        // check for max-width style
+                                        var maxWidthIdx = attr[1].indexOf("max-width");
+                                        if (maxWidthIdx >= 0) {
+                                            widthStyles += (minWidthIdx >= 0 ? " " : "") + attr[1].substring(maxWidthIdx, attr[1].indexOf(";", maxWidthIdx)+1);
```

Comment:
Instead of `(minWidthIdx >= 0 ? " " : "")` I think you should do `(widthStyles.length > 0 ? " " : "")`.

Although I think it might be better if `widthStyles` is an array, then in the end you would check for its length, and also would do `widthStyles.join(" ")` if you really want space between your rules.

P.S. I don't think space between rules are actually necessary. So I would just get rid of the check altogether and add the rules after each other without any space in between.